### PR TITLE
Merge pull request #1 from AbhinavRobinson/master

### PR DESCRIPTION
**Pull request by aniketbiprojit**
Update Deno Build

### CODE REVIEW

**Comment by aniketbiprojit** [comment]
@AbhinavRobinson

**Comment by aniketbiprojit** [comment]
Code formatting done

**Comment by aniketbiprojit** [comment]
Add label hacktoberfest-accepted to the PR please.

https://hacktoberfest.digitalocean.com/hacktoberfest-update

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+	"jsxSingleQuote": true,
```
Comment (AbhinavRobinson) [inline .prettierrc:2]: These settings are pretty okay.

**Review by AbhinavRobinson — COMMENTED** [review]
Good Settings may use later.

**Pull request merged**